vncviewer/DesktopWindow.cxx

```diff
@@ -322,6 +332,25 @@ void DesktopWindow::setCursor(int width, int height,
 }
 
 
+void DesktopWindow::setCursorPos(const rfb::Point& pos)
+{
```

Comment:
i take the mouse-grabbing comment back, this worked!

Reply:
The grab check was fine and much welcome. My point was to **also** check focus. But then again, I guess grab implies focus so I guess that isn't needed...

Reply:
Ah! got it, i didn't quite grok the meaning of the other comment. Restored the grab check.

Reply:
_really_ restored the grab check.